### ISSUE TITLE
[Bug] Low Codex

### ISSUE THREAD

**Issue by ZachZent** [tsunamayo/Skywanderers#1770]
The codex is rather low on the screen. You are also unable to drag the codex around on the screen, though I'm not sure if we were able to do that in the first place.

![](https://cdn.discordapp.com/attachments/406664114249793536/700555401086763039/unknown.png)

**Comment by tsunamayo** [comment]
Yes I made the change so that the things fit on a 720p monitor. Below it rescale. The codex should be attached to the center, I will see if I can make something smarter about that. What is your screen resolution?
No I did not coded the dragging feature, it is not an MMO ;).
Cheers
